Task1_KFox/app/index.html

```diff
@@ -7,142 +7,187 @@
         <link rel="stylesheet" href="css/styles.css">
     </head>
     <body>
```

Comment:
Please specify roles everywhere within the page.
http://accessibleculture.org/research-files/ozewai2011/images/html5AriaTestCase.png